public/javascripts/controllers/videoUpload.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * jQuery File Upload Plugin JS Example
+ * https://github.com/blueimp/jQuery-File-Upload
+ *
+ * Copyright 2010, Sebastian Tschan
+ * https://blueimp.net
+ *
+ * Licensed under the MIT license:
+ * https://opensource.org/licenses/MIT
+ */
+
+/* global $, window */
+
+$(function () {
+    'use strict';
+
+    // Initialize the jQuery File Upload widget:
+    $('#fileupload').fileupload({
```

Comment:
There are three declarations of `$('#fileupload')`, remove the irrelevant ones, also the commented code we don't need

Reply:
changed